tmcprototype/SubarrayNode/SubarrayNode/SubarrayNode.py

```diff
@@ -517,19 +527,31 @@ def Scan(self, argin):
         try:
             print(CONST.STR_SCAN_IP_ARG, argin)
             scanParameter = json.loads(argin[0])
-            scanDuration = str(scanParameter["scanDuration"])
 
-            scanTimestamp = "0"
-            if type(float(scanTimestamp)) == float:
-                assert self._obs_state != CONST.ENUM_SCANNING, CONST.SCAN_ALREADY_IN_PROGRESS
-                print(CONST.STR_GRP_DEF, self._dish_leaf_node_group.get_device_list())
-                self._read_activity_message = CONST.STR_SCAN_IP_ARG + str(scanTimestamp)
-                self._read_activity_message = CONST.STR_GRP_DEF + str(
-                    self._dish_leaf_node_group.get_device_list())
-                # set obsState to SCANNING when the scan is started
-                self._obs_state = CONST.ENUM_SCANNING
-                self.set_status(CONST.STR_SA_SCANNING)
-                self.dev_logging(CONST.STR_SA_SCANNING, int(tango.LogLevel.LOG_INFO))
+            if self._obs_state == CONST.ENUM_READY:
+                # assert self._obs_state != CONST.ENUM_SCANNING, CONST.SCAN_ALREADY_IN_PROGRESS
```

Comment:
Remove commented code.

Reply:
Code is already updated with this changes.